runtime/LICENSE

```diff
@@ -0,0 +1,19 @@
+Copyright © 2022 INRIA, CNRS and contributors
```

Comment:
Just do not put any year. I know it's not conventional, but there is no legal obligation to follow the convention here. No year means no update ever necessary.